components/layout/header.vue

```diff
@@ -69,23 +68,46 @@
             </div>
           </div>
           <div class="py-6 px-5 space-y-6">
-            <div class="grid grid-cols-2 gap-y-4 gap-x-8">
+            <div class="flex flex-col float-right space-y-4">
               <NuxtLink
                 v-for="item in navigation"
                 :key="item.name"
                 :to="item.to"
-                class="text-base font-medium text-gray-900 hover:text-gray-700"
+                class="text-bas font-light text-gray-900 hover:text-gray-700"
               >
                 {{ item.name }}
               </NuxtLink>
-            </div>
-            <div>
-              <a
-                href="/request"
-                class="w-full flex items-center justify-center px-4 py-2 border border-transparent rounded-md shadow-sm text-base font-medium text-white bg-indigo-600 hover:bg-indigo-700"
-              >
-                Need help?
-              </a>
+              <div class="m-6">
+                <select
+                  class="form-select appearance-none rounded-lg block w-full px-3 py-1.5 text-base font-normal text-gray-700 bg-white bg-clip-padding bg-no-repeat border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+                  id="language"
+                >
+                  <option selected>EN</option>
+                  <option>GER</option>
+                  <option>RUS</option>
+                  <option>UKR</option>
+                </select>
+              </div>
+              <br />
+              <div class="h-20"></div>
+              <br />
+              <div class="float-right m-8 space-y-2">
+                <a href="https://t.me/wehelplive">
+                  <Icon icon="akar-icons:telegram-fill" height="24" width="24"
```

Comment:
In the footer, I use another icon for social networks. How we can make them consistent?

Reply:
I looked into the footer code, I think this one is a lot simpler to use. What say?